conf.d/python.d/phpfpm.conf

```diff
@@ -0,0 +1,4 @@
+update_every: 3
```

Comment:
ideally, I would love to have `update_every` on every module unset, to let it inherit the `update_every` supplied by netdata